docs/reference/cluster/reroute.asciidoc

```diff
@@ -68,3 +68,5 @@ The commands supported are:
     allocate the shard to. It also accepts `allow_primary` flag to
     explicitly specify that it is allowed to explicitly allocate a primary
     shard (might result in data loss).
+    
+WARNING: Using `allow_primary` in conjunction with `allocate` command to force a primary shard allocation, will cause data contained in existing targeted shard to be lost. Use with caution. 
```

Comment:
Do you think the warning tag is too much? We've seen people delete their
data multiple times not realizing that flag can be destructive.

On 4 August 2015 at 21:46, Nik Everett <notifications@github.com> wrote:

> In docs/reference/cluster/reroute.asciidoc
> <https://github.com/elastic/elasticsearch/pull/12503#discussion_r36232548>
> :
>
> > @@ -68,3 +68,5 @@ The commands supported are:
> >      allocate the shard to. It also accepts `allow_primary` flag to
> >      explicitly specify that it is allowed to explicitly allocate a primary
> >      shard (might result in data loss).
> > +
> > +WARNING: Using `allow_primary` in conjunction with `allocate` command to force a primary shard allocation, will cause data contained in existing targeted shard to be lost. Use with caution.
>
> I do like Clint's wording better. If he had fewer commas I'd merge his
> wording.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/elasticsearch/pull/12503/files#r36232548>.
>